en/frames_rover/ackermann_rover.md

```diff
@@ -88,7 +62,7 @@ To get an overview of all parameters that are related to the Ackermann rover mod
 
 ### General Parameters
 
-These parameters affect the general behaviour of the rover. This will influence both auto and manual modes.
+These parameters affect the general behavior of the rover. This will influence both auto and manual modes.
 
 | Parameter                                                                                       | Description                                | Unit |
 | ----------------------------------------------------------------------------------------------- | ------------------------------------------ | ---- |
```

Comment:
The following section is "Slew rates". This says "The slew rates are not based on measurements but on assumed linear relation between the throttle".

This tells me what they are, but not what I should set them to. How do I go about deciding what they should be. Say assuming my vehicle can go from 0 to 60km/hour in 5 seconds, and stay on the road if doing a circle with radius X, at Y m/s.

Reply:
Similarly for `RA_MAX_SPEED` - is the approach to just set this impossibly high, drive the vehicle as fast as it can go, and put in the top value?

Where possible we need to be instructional - "this is what you do". The parameter reference is informational "this is what it is"

Reply:
This is ^^^ an important point. When you come to this again, look at every section and ask "what am I asking the user to actually _do_.